src/hotspot/cpu/aarch64/vm_version_aarch64.cpp

```diff
@@ -703,6 +714,7 @@ void VM_Version::initialize_cpu_information(void) {
   get_compatible_board(_cpu_desc + desc_len, CPU_DETAILED_DESC_BUF_SIZE - desc_len);
   desc_len = (int)strlen(_cpu_desc);
   snprintf(_cpu_desc + desc_len, CPU_DETAILED_DESC_BUF_SIZE - desc_len, " %s", _features_string);
+  fprintf(stderr, "_features_string = \"%s\"", _features_string);
```

Comment:
Was this line added by mistake? Looks like a leftover.

Reply:
@iwanowww  Oooops, yes. Addressed in [https://github.com/openjdk/jdk/pull/24717](url)  together with another forgotten (though not strictly necessary) change. Thanks for catching it!
Could you review that (really short) PR?

Reply:
@ferakocz That link has the right label but leads to the wrong place. It should point [here](https://github.com/openjdk/jdk/pull/24717).

Reply:
@iwanowww Thanks for the heads up!